rand_distr/src/lib.rs

```diff
@@ -107,12 +107,14 @@ pub use self::unit_disc::UnitDisc;
 pub use self::unit_sphere::UnitSphere;
 pub use self::weibull::{Error as WeibullError, Weibull};
 #[cfg(feature = "alloc")]
-pub use self::weighted::{WeightedError, WeightedIndex};
+pub use rand::distributions::weighted::{WeightedError, WeightedIndex};
+#[cfg(feature = "alloc")]
+pub use weighted_alias::WeightedAliasIndex;
```

Comment:
Unfortunately this results in `WeightedAliasIndex` being shown as re-exported item in documentation.